lib/Conversion/TorchToTcp/Misc.cpp

```diff
@@ -84,7 +86,8 @@ class ConvertAtenBroadcastToOp : public OpConversionPattern<AtenBroadcastToOp> {
       int64_t staticDimSize;
       if (i < newLeadingDims ||
           !matchPattern(newDimSize, m_TorchConstantInt(&staticDimSize)) ||
-          staticDimSize != -1) {
+          (staticDimSize != -1 &&
```

Comment:
nit: can we use `ShapedType::kDynamic` instead of `-1` here?
```
(staticDimSize != ShapedType::kDynamic &&
...
```

Reply:
💯 didn't know about that

Reply:
As discussed offline, please ignore since this condition is to check for `-1` in `torch.broadcast_to` or `torch.expand` which stands for "unchanged dim".